### DISCUSSION TITLE
使用waline评论产生跨域问题怎么解决呀？

### DISCUSSION THREAD

**Discussion by bobtmee** [walinejs/waline#1578] [Q&A] [answered]
[vuepress-theme-hope](https://vuepress-theme-hope.github.io/v2/zh/)
搜到文档说不加/ 我是没有加的  并且是在本地开发环境中遇到的

主题配置 
      /**
       * Using Waline
       */
      provider: "Waline",
      serverURL: "http://waline.ss.com",

控制台报错
Access to fetch at 'http://waline.ss.com/comment?path=%2Fzh%2Fdemo%2F&pageSize=10&page=1&lang=zh-CN&sortBy=insertedAt_desc' from origin 'http://192.168.3.113:8080' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
Waline.vue:322 GET http://waline.ss.com/comment?path=%2Fzh%2Fdemo%2F&pageSize=10&page=1&lang=zh-CN&sortBy=insertedAt_desc net::ERR_FAILED

我使用的Vercel 部署waline (服务端)    直接打开是没有问题的 但是引入到主题  就产生跨域错误了

**Comment by lizheming** [comment, marked as answer]
这是什么玩意
![image](https://user-images.githubusercontent.com/424491/201275072-a2573784-cbe8-4e8d-9dba-363d152fe192.png)

**Reply by bobtmee** [reply]
修改了 现在可以打开了  单独使用是没有错误的，  但是在vuepress-theme-hope 中使用 报跨域错误   这个是我本地开发的时候报的错误  没有线上地址 ，麻烦老大看看

**Reply by lizheming** [reply]
提供下地址，不然我没法看

**Reply by bobtmee** [reply]
谢谢老大回复 经过测试发现部署到线上不存在该问题了
